### DISCUSSION TITLE
Ensure GitJournal works with gh-pages

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#777] [Feature Requests]
Test out gh-pages and document what features GitJournal might be missing for it to work properly. Also document this on the website.

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by cvk** [comment]
Getting this to work with clear setup instructions could be a big win.

**Comment by vHanda** [comment]
I just tried it out, and it seems to work perfectly!

**Comment by cvk** [comment]
I understand you want charge for YAML gui editing as a premium feature. But with YAML disabled in the settings my front matter shouldn't be touched at all. Stripped titles is not expected behavior from the user perspective with it disabled. Hoping this is a bug.

**Comment by vHanda** [comment]
> I understand you want charge for YAML gui editing as a premium feature. But with YAML disabled in the settings my front matter shouldn't be touched at all. Stripped titles is not expected behavior from the user perspective with it disabled. Hoping this is a bug.

I don't follow. Could you please give me an example?

**Comment by cvk** [comment]
![gitjournal-edit](https://user-images.githubusercontent.com/727/90595339-7d83df00-e1a1-11ea-95c6-c7c907f6353f.png)

**Comment by vHanda** [comment]
Definitely a bug!! I'll fix it today
